src/Conn.cc

```diff
@@ -341,12 +344,15 @@ RecordVal* Connection::BuildConnVal()
 		conn_val = new RecordVal(connection_type);
 
 		TransportProto prot_type = ConnTransport();
-
 		RecordVal* id_val = new RecordVal(conn_id);
 		id_val->Assign(0, new AddrVal(orig_addr));
 		id_val->Assign(1, val_mgr->GetPort(ntohs(orig_port), prot_type));
 		id_val->Assign(2, new AddrVal(resp_addr));
 		id_val->Assign(3, val_mgr->GetPort(ntohs(resp_port), prot_type));
+		cout << conn_id_type << ";" << vlan << endl;
+		if (conn_id_type == BASIC_WITH_VLAN && vlan != 0) {
+			id_val->Assign(4, val_mgr->GetInt(vlan));
+		}
```

Comment:
Leftover debug output and wrong bracing/whitespace style (see https://docs.zeek.org/en/latest/devel/style_guide.html)